setup.py

```diff
@@ -29,6 +29,7 @@ def read(fname):
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python",
         'Programming Language :: Python :: 3.5',
```

Comment:
Do we want to keep this line for this version of the container? Will it still run with Python 3.5?

Reply:
Yes, I think we want to keep 3.5 for this commit because at least the SXC module will work with 3.5. But we will have to drop 3.5 in the next commit when we install tornasole.

Reply:
Ah, got it.